packages/storage/amplify_storage_s3_dart/lib/src/model/s3_item_with_access_level.dart

```diff
@@ -48,9 +48,9 @@ class S3ItemWithAccessLevel extends StorageItemWithAccessLevel<S3Item> {
           targetIdentityId: targetIdentityId,
         );
 
-  /// The identity id of another user who uploaded the containing `storageItem`.
+  /// The identity id of another user who uploaded the object that is
+  /// represented by the containing `storageItem`.
```

Comment:
Any in-scope items can be linked to

```suggestion
  /// represented by the containing [storageItem].
```